test/jdk/com/sun/jdi/lib/jdb/Debuggee.java

```diff
@@ -127,33 +126,38 @@ public ProcessBuilder prepare() {
         }
 
         public Debuggee launch(String name) {
-            return new Debuggee(prepare(), name, waitForPortPrint, expectedOutputBeforeThrow);
+            return new Debuggee(prepare(), name,
+                onthrow.isEmpty() ?
+                JDWP::parseListenAddress :
+                Launcher::parseLaunchEchoListenAddress
```

Comment:
Please increase indent for conditional operator
```suggestion
                onthrow.isEmpty() ?
                    JDWP::parseListenAddress :
                    Launcher::parseLaunchEchoListenAddress
```
or
```suggestion
                onthrow.isEmpty()
                    ? JDWP::parseListenAddress
                    : Launcher::parseLaunchEchoListenAddress
```